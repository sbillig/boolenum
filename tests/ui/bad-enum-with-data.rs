use boolenum::BoolEnum;

#[derive(BoolEnum)]
pub enum EnumWithData {
    Yes(String),
    No,
}

fn main() {}
