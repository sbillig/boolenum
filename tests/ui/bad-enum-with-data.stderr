error: BoolEnum can only be used on enums with unit-like variants

  = help: try `enum EnumWithData { No, Yes }`

 --> $DIR/bad-enum-with-data.rs:5:5
  |
5 |     Yes(String),
  |     ^^^
